generator/_assets/styles/less/article.less

```diff
@@ -110,10 +110,13 @@ article {
 
     .article_title {
       display: flex;
-      align-items: center;
+      align-items: flex-start;
+      word-wrap: break-word;
+      grid-gap: 12px;
 
       h1 {
         flex-grow: 1;
+        max-width: calc(100% - 110px);
```

Comment:
in the phone-down this should be changed to 100%, otherwise on the mobile biew will look wrong

Reply:
updated commit